x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -36,9 +35,9 @@ import {
 import { ExplorerState } from './reducers/explorer_reducer';
 import { hasMatchingPoints } from './has_matching_points';
 import { ExplorerNoInfluencersFound } from './components/explorer_no_influencers_found/explorer_no_influencers_found';
```

Comment:
Not part of the changes for this PR, but we should change the 'no results' found when viewing by job ID, to say something like `No results found for selected job` / `No results found for selected jobs` as this swim lane is showing the bucket scores for the jobs.

![image](https://user-images.githubusercontent.com/7405507/86234298-9a7f3700-bb8e-11ea-8592-087e92c6a77c.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/70063/commits/04bf4ca21a113892b23c4d5391f84833beb71b25